engine/src/test/java/com/google/android/fhir/db/impl/dao/LocalChangeUtilsTest.kt

```diff
@@ -0,0 +1,183 @@
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.db.impl.dao
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.android.fhir.db.impl.dao.LocalChangeUtils.mergeLocalChanges
+import com.google.android.fhir.db.impl.entities.LocalChangeEntity
+import com.google.android.fhir.toTimeZoneString
+import java.util.Date
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+/** Unit tests for [LocalChangeUtils]. */
+@RunWith(RobolectricTestRunner::class)
+class LocalChangeUtilsTest {
+  val payload1 =
+    """
+     {
+       "resourceType": "Patient",
+       "id": "human",
+       "name": [
+         {
+           "use": "usual",
+           "given": [
+             "Kenzi"
+           ]
+         }
+       ]
+     }
+     """
+
+  val json_patch =
+    """
+       [
+         {
+          "op": "replace",
+          "path": "/name/0/use",
+          "value":"Ana2k"          
+          }
+       ]
+     """.trimMargin()
+
+  // converting the json payload to json node type
+  val objectMapper = ObjectMapper()
+  val payload_json_node: JsonNode = objectMapper.readTree(payload1)
+
+  val insert_payload1 =
+    LocalChangeEntity(
+      id = 1L,
+      resourceType = "Test1",
+      resourceId = "Anam1",
+      timestamp = Date().toTimeZoneString(),
+      payload = "payload1",
+      type = LocalChangeEntity.Type.INSERT
+    )
+  val insert_payload2 =
+    LocalChangeEntity(
+      id = 2L,
+      resourceType = "Test2",
+      resourceId = "Anam2",
+      timestamp = Date().toTimeZoneString(),
+      payload = "payload2",
+      type = LocalChangeEntity.Type.INSERT
+    )
+  val delete_payload =
+    LocalChangeEntity(
+      id = 3L,
+      resourceType = "Test3",
+      resourceId = "Anam3",
+      timestamp = Date().toTimeZoneString(),
+      payload = "payload3",
+      type = LocalChangeEntity.Type.DELETE
+    )
+  val update_json_patch =
+    LocalChangeEntity(
+      id = 4L,
+      resourceType = "Test4",
+      resourceId = "Anam4",
+      timestamp = Date().toTimeZoneString(),
+      payload = json_patch,
+      type = LocalChangeEntity.Type.UPDATE
+    )
+  val insert_payload_json_node =
+    LocalChangeEntity(
+      id = 5L,
+      resourceType = "Test5",
+      resourceId = "Anam5",
+      timestamp = Date().toTimeZoneString(),
+      payload = payload_json_node.toString(),
+      type = LocalChangeEntity.Type.INSERT
+    )
+
+  @Test
+  fun response_test_insert_payload1_insert_payload2() {
```

Comment:
```suggestion
  fun mergeLocalChanges_insertTwice() {
```

same with the other names

Reply:
@jingtang10  @vincent-karuri I have implemented all the required changes. Kindly review this pr. Thanks!